### DISCUSSION TITLE
Docker Desktop error

### DISCUSSION THREAD

**Discussion by SisiVero** [camicroscope/caMicroscope#690] [Q&A]
I have successfully cloned the Distro repo and installed necessary packages. But I've kept encountering this error each time I try to start my Docker desktop. I've researched and researched but solutions I find do not help me. Is anyone having the same issue? This is Docker for windows 10. I downloaded. Is there something I'm not doing right?
![docker error](https://github.com/camicroscope/caMicroscope/assets/105162163/7d5827a7-85e3-4046-b7e4-83b304e4b997)

**Comment by SisiVero** [comment]
Incase you encounter this error and you're using ho Pavilion, your BIO key is "f10".
Restart your computer and keep pressing f10 until your advanced settings shows up. Then navigate to Cpu settings and enable virtualization.

**Comment by birm** [comment]
I see two different install paths for windows (https://docs.docker.com/desktop/install/windows-install/). One seems to require WSL (https://learn.microsoft.com/en-us/windows/wsl/install) 

If you do figure out what needs to be installed/enabled, please update!

**Reply by SisiVero** [reply]
Hi @birm, virtualization needs to be enabled for it to work.
If virtualization is not enabled in your machine, docker will not run.
Here I gave a full answer on that and one of us found it useful. Here is the link to tell solution: https://github.com/orgs/camicroscope/discussions/685#discussioncomment-8710259
